### ISSUE TITLE
YTKillerPlus is crashing.

### ISSUE THREAD

**Issue by asdfzxcvbn** [iKarwan/YTKillerPlus#9]
When sideloading (no TrollStore), the app crashes. This is likely a problem with the IPA.

**Comment by Mighel881** [comment]
Crash crash crash YouTube killer and pro version not work with 18.7.05  on update

**Comment by asdfzxcvbn** [comment]
the app won't even open with the latest update! you get an integrity error. i've tried with 3 different WORKING certificates.

**Comment by badgerbrownie** [comment]
Just downloaded at installed via TrollStore and the app won’t open. Crashes immediately upon opening. I’m on 14.7.1

**Comment by asdfzxcvbn** [comment]
yeah, every release doesn't work. and the fact that the dev is just ignoring this isn't good. i would suggest requesting a refund or maybe going after another mod. appdb has a bunch of them, anyway.

**Comment by iKarwan** [comment]
it's clearly says in the description of the tweak not support palra1nn nor xina jailbreaks pls read the description
i'm not ignoring it as everything is in the description of the tweak and write noted to everyone those jailbreaks not supporting the payment api  and u need to usse the ipa or trollstore version with removeng the deb

**Comment by iKarwan** [comment]
> it's clearly says in the description of the tweak not support palra1n nor xina jailbreaks pls read the description i'm not ignoring it as everything is in the description of the tweak and write noted to everyone those jailbreaks not supporting the payment api and u need to use the ipa or trollstore version with removing the deb

Activation NOTICE!!:
· NOTE!! 1- you need to activate the license with in 20 seconds otherwise the app will crash as it normal due to the security included .
· NOTE!! 2- the activation screen may appears sometimes when u rejailbreak or reboot the device, due to security & anti crack included | you just need to enter the code again and click on reset code then activate .
· NOTE!! 3- not supporting proxy or VPN if they on the app will crash on launch .
· NOTE!! 4- if u on palra1n or XinaA15 Jailbreak please use the iPA or TrollStore version if u have freeze on YouTube first launch .

**Comment by badgerbrownie** [comment]
> > it's clearly says in the description of the tweak not support palra1n nor xina jailbreaks pls read the description i'm not ignoring it as everything is in the description of the tweak and write noted to everyone those jailbreaks not supporting the payment api and u need to use the ipa or trollstore version with removing the deb
> 
> Activation NOTICE!!: · NOTE!! 1- you need to activate the license with in 20 seconds otherwise the app will crash as it normal due to the security included . · NOTE!! 2- the activation screen may appears sometimes when u rejailbreak or reboot the device, due to security & anti crack included | you just need to enter the code again and click on reset code then activate . · NOTE!! 3- not supporting proxy or VPN if they on the app will crash on launch . · NOTE!! 4- if u on palra1n or XinaA15 Jailbreak please use the iPA or TrollStore version if u have freeze on YouTube first launch .
> 
> after removing the deb..
> 
> if u still have issue please contact via @ikg_hd on twitter or ikghd3@gmail ..

Thank you for the replies here and on Twitter.

**Comment by asdfzxcvbn** [comment]
i am not jailbroken.

**Comment by asdfzxcvbn** [comment]
just tried again, NOT jailbroken and NO vpns/proxies on, and it's still crashing a bit after launching. i dont even reach the activation screen.

**Comment by asdfzxcvbn** [comment]
still doesn’t work. seriously don’t know anything about this?

**Comment by badgerbrownie** [comment]
> still doesn’t work. seriously don’t know anything about this?

I'm still having issues as well. Any chance you can work on an update please?

**Comment by asdfzxcvbn** [comment]
this has been happening for a looong time now. the dev just keeps updating the readme with new releases and doesnt do anything.

**Comment by asdfzxcvbn** [comment]
however youtubemurderer on appdb is working right now
